### PR TITLE
fix: Template editor tweaks

### PR DESCRIPTION
**Pull request by jekabs-karklins**
## Description

Miscellaneous Template Editor UI tweaks
Peek 2021-09-10 14-39

## Motivation and Context

Making UI consistent and more responsive

## How Has This Been Tested

-[x] E2E

## Fixes

https://github.com/UserOfficeProject/user-office-frontend/issues/527

## Depends on

N/A

### CODE REVIEW

**Review by martin-trajanovski — APPROVED** [review]
Some better space usage! Looks great!

File: src/components/template/TemplateTopicEditor.tsx [hunk 1]
```diff
@@ -187,60 +213,53 @@ export default function QuestionaryEditorTopic(props: {
...
-          ) : (
-            <div>
-              <span data-cy="topic-title">{values.title}</span>{' '}
-              <EditIcon
-                fontSize="small"
-                data-cy="topic-title-edit"
-                onClick={() => {
-                  setIsEditMode(true);
+            <div style={{ display: 'flex' }}>
+              <Field
```
Comment (TCMeldrum) [inline src/components/template/TemplateTopicEditor.tsx:217]: Changes look good! Had to look over with the [Wave tool](https://wave.webaim.org/) for any accessibility concerns and it showed a missing form label when editing a topic name would you be able to add one with aria-label to this field maybe?

**Review by TCMeldrum — APPROVED** [review]
Cool thanks for adding that label! Changes look good!

File: src/components/template/TemplateTopicEditor.tsx (hunk 1 above)
Comment (jekabs-karklins) [reply src/components/template/TemplateTopicEditor.tsx:220]: Good point I added the label for the screen readers

**Pull request merged**